reV/bespoke/bespoke.py

```diff
@@ -277,6 +303,64 @@ def _parse_output_req(self):
             logger.error(msg)
             raise KeyError(msg)
 
+    def _parse_prior_run(self):
+        """Parse prior bespoke wind plant optimization run meta data and make
+        sure the SAM system inputs are set accordingly."""
+
+        # {meta_column: sam_sys_input_key}
+        required = {'capacity': 'system_capacity',
+                    'turbine_x_coords': 'wind_farm_xCoordinates',
+                    'turbine_y_coords': 'wind_farm_yCoordinates'}
+
+        if self._prior_meta:
+            missing = [k for k, _ in required.items() if k not in self.meta]
```

Comment:
Just curious, why not simply `missing = [k for k in required if k not in self.meta]` here?

Reply:
Oh you know what, i was doing in required.keys() but pylint got angry and i think i misinterpreted the message as "use items" instead of "iterate directly"